monkeytype/cli.py

```diff
@@ -133,7 +141,9 @@ def apply_stub_handler(args: argparse.Namespace, stdout: IO, stderr: IO) -> None
             src_path
         ])
         try:
-            subprocess.run(cmd, shell=True, check=True, stderr=subprocess.STDOUT, stdout=subprocess.PIPE)
+            proc_output = subprocess.run(cmd, shell=True, check=True, stderr=subprocess.STDOUT, stdout=subprocess.PIPE)
+            for line in proc_output.stdout.decode('utf-8').splitlines():
```

Comment:
This is great to have for error/warning cases, but does retype have any output in the success case? Can you paste what it is, if so? Just don't want to make things too noisy.

Also, there's no reason to splitlines and then print each line, when we could get the same effect by just printing the whole thing.

Actually, couldn't we just set `stdout=subprocess.STDOUT` above (instead of pipe) and get the same effect?

Reply:
Yes i'll simplify the printing. I think it evolved that way through iterations and I didn't notice the splitlines+loop.

The success case has no output.

If I set stdout=subprocess.STDOUT, I get an exception:
OSError: [Errno 9] Bad file descriptor

Reply:
Oh, maybe `stdout=sys.stdout`? If that doesn't work, then yeah just `print(proc_output.stdout)`